appspec-scripts/after_install.sh

```diff
@@ -1,20 +1,35 @@
 #!/bin/bash
 set -e
-export PATH=$PATH:/home/ubuntu/.cargo/bin
 
+TAG=$(curl http://169.254.169.254/latest/meta-data/tags/instance/stable)
 APP_NAME=$(echo $APPLICATION_NAME | awk -F- '{ print $2 }')
  
-# Prepare directory
-sudo rm -rf ~/$APP_NAME/tlsn
-sudo mv ~/tlsn/ ~/$APP_NAME
-sudo mkdir -p ~/$APP_NAME/tlsn/notary-server/target/release
-sudo chown -R ubuntu.ubuntu ~/$APP_NAME
- 
-# Download .git directory
-aws s3 cp s3://tlsn-deploy/$APP_NAME/.git ~/$APP_NAME/tlsn/.git --recursive
+if [ $APP_NAME = "stable" ]; then
+  # Prepare directories for stable versions
+  sudo mkdir ~/${APP_NAME}_${TAG}
+  sudo mv ~/tlsn ~/${APP_NAME}_${TAG}
+  sudo mkdir -p ~/${APP_NAME}_${TAG}/tlsn/notary-server/target/release
+  sudo chown -R ubuntu.ubuntu ~/${APP_NAME}_${TAG}
+
+  # Download .git directory
+  aws s3 cp s3://tlsn-deploy/$APP_NAME/.git ~/${APP_NAME}_${TAG}/tlsn/.git --recursive
```

Comment:
so s3 will always only contain the latest version of binary and .git right?

Reply:
Correct! Since the workflow execution is performed sequentially, each new workflow execution, will upload the latest version to S3.